content/1.7/back-office/dashboard/dashboard.md

```diff
@@ -0,0 +1,179 @@
+
+ - **Sales** - displays the sales amount in the default currency tax excluded within the date range. A tooltip is displayed: "Sum of revenue (excl. tax) generated within the range by orders considered validated.".
+ - **Orders** - displays the number of orders within the date range. A tooltip is displayed: "Total number of orders received within the date range that are considered validated.".
+ - **Cart Value** - displays the average cart value in the default currency tax excluded within the date range. A tooltip is displayed: "Average Cart Value is a metric representing the value of an average order within the date range. It is calculated by dividing Sales by Orders.".
+ - **Visits** - displays the number of visits within the date range. A tooltip is displayed: "Total number of visits within the date range. A visit is the period of time a user is actively engaged with your website.".
+ - **Conversion Rate** - displays the conversion rate in percentage. A tooltip is displayed: "Ecommerce Conversion Rate is the percentage of visits that resulted in an validated order.".
+ - **Net Profit** - displays the net profit in the default currency tax excluded. A tooltip is displayed: "Net profit is a measure of the profitability of a venture after accounting for all Ecommerce costs. You can provide these costs by clicking on the configuration icon right above here.".
+
+After selecting a tab, the corresponding graphic is displayed. When hovering the graphic with the mouse, the details are displayed.
+Additionally, when comparing two time periods, it displays the evolution of each figure in percentages.
+
+## Left column
+
+### Activity overview (Dashboard Activity module)
+
+#### Call to action 
+
+On the top right of this block there are 2 buttons:
+
+ - **Configure** - when pressed, changes the activity overview to a configuration block with the following field:
+    - **Active cart** - drop down field with options: 15, 30, 45, 60, 90, 120.
+    - **Online visitors** - drop down field with options: 15, 30, 45, 60, 90, 120.
+    - **Abandoned cart (min)** - input field measured by "d".
+    - **Abandoned cart (max)** - input field measured by "d".
+    - "Save" button, when pressed - reverts back to the dashboard activity block.
+ - **Refresh** - refreshes the stats of the Activity Overview block.
+
+Below the following metrics are displayed:
+
+ - **Online visitors**
+ - **Active shopping carts**
+ - **Currently Pending**
+    - **Orders** - when pressed reditrects to the Orders page.
+    - **Return/Exchanges** - when pressed, redirects to the Merchandise Returns page.
+    - **Abandoned Carts** - when pressed, redirects to the Shopping Carts page.
+    - **Out of stock products** - when pressed, redirects to the Monitoring page.
+ - **Notifications**
+    - **New Messages** - when pressed, redirects to the Customer Service page.
+    - **Product Reviews** - when pressed, redirects to the Configure page.
```

Comment:
I don't understand which configure page you are talking? Could you bring more precision?

Reply:
This is for the Activity overview, firstly the configure button when pressed what is showed, and trhe first view of the fields as well.

Reply:
See my suggestion, it was not very clear :)